### DISCUSSION TITLE
Multi GPU training

### DISCUSSION THREAD

**Discussion by ramchandracheke** [Project-MONAI/MONAI#2265] [Q&A] [answered]
Hi,

Can you please tell me that does it support multi-GPU? and How to use multiGPU set-up?

Thanks a million!

**Comment by ramchandracheke** [comment, marked as answer]
I found the solution at https://pytorch.org/tutorials/beginner/blitz/data_parallel_tutorial.html

**Comment by Nic-Ma** [comment]
Hi @ramchandracheke ,

MONAI can support multi-gpu, you can check the PyTorch tutorial and use similar logic with MONAI components.

Thanks.
